tests/dind.sh

```diff
@@ -1,15 +1,20 @@
-#!/bin/sh
+#!/bin/bash
 
+K3D_EXE="/bin/k3d"
 K3D_IMAGE_TAG=$1
 
 k3de2e=$(docker run -d --rm \
           -v "$(pwd)"/tests:/tests \
           --privileged \
-          -e EXE="/bin/k3d" \
+          -e EXE="$K3D_EXE" \
           -e CI="true" \
+          --name "k3d-e2e-runner" \
```

Comment:
This could potentially lead to naming conflicts on dev machines (probably unlikely, but possible).
Only using the unique ID (assigned to `$k3de2e`) or appending a timestamp is probably safer.

Reply:
Ok, I will add a timestamp `TIMESTAMP=$(date "+%H%M%S")`